modules/stochastic_tools/include/samplers/CartesianProductSampler.h

```diff
@@ -0,0 +1,43 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "Sampler.h"
+
+class CartesianProductSampler;
```

Comment:
So is CartesianProduct the generally accepted name instead of Grid?  It looks like you're using the 'grid' name in the testing.

Reply:
Yes, "Cartesian Product" is the actual name. I will update the stuff to use that name.